media/static-bundles.json

```diff
@@ -1558,6 +1558,12 @@
       ],
       "name": "product_pocket"
     },
+    {
+      "files": [
+        "js/externalpages/pocket/mobile-nav.es6.js"
+      ],
+      "name": "pocket-about"
```

Comment:
I'm guessing this nav will end up on more pages - is it worth naming the bundle to something more generic, like `pocket-nav`?

Reply:
Good point!